src/MassTransit.Analyzers/MassTransit.Analyzers.csproj

```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.3.0" PrivateAssets="all" />
```

Comment:
Is this absolutely necessary? I believe it breaks older versions of Visual Studio.

Reply:
As mention by @RemcoBlok:

> So, if you want to analyze C#10 and .NET 6 code the minimum version of the .NET compiler platform package version referenced should be 4.0.

But the tests are working without it.
I've trying on my work project, and it doesn't change anything on it.
So... I don't know 😆

Reply:
If it works without it (staying with 3.8) then let's keep it at 3.8. It's backwards compatible and will work with the newest versions. I'm pretty sure I rolled it back from 4.3 once previous due to issues.

Reply:
Done.